zstacklib/zstacklib/network/ovs/controller.py

```
@@ -0,0 +1,502 @@
+            shell.run(CTL_BIN + '--no-wait remove Open_vSwitch . other_config pmd-cpu-mask')
+        else:
+            shell.run(CTL_BIN + f'--no-wait set Open_vSwitch . other_config:pmd-cpu-mask={cpu_mask}')
+            if not self.ovs.check_ovs_configuration('pmd-cpu-mask', cpu_mask):
+                raise OvsError('Config pmd cpu mask for ovs failed.')
+
+    @linux.retry(times=3, sleep_time=1)
+    def config_lacp_fallback_ab_for_ovs(self) -> None:
+        """Configure LACP fallback for OVS."""
+        shell.run(CTL_BIN + '--no-wait set Open_vSwitch . other-config:lacp-fallback-ab=true')
+        if not self.ovs.check_ovs_configuration('lacp-fallback-ab', 'true'):
+            raise OvsError('Config lacp fallback ab for ovs failed.')
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)